client/blocks/google-my-business-stats-nudge/index.js

```diff
@@ -0,0 +1,116 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+import { flow } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import { recordTracksEvent } from 'state/analytics/actions';
+import SectionHeader from 'components/section-header';
+import QueryPreferences from 'components/data/query-preferences';
+import { isGoogleMyBusinessStatsNudgeDismissed } from 'state/selectors';
+import { dismissNudge } from './actions';
+
+class GoogleMyBusinessStatsNudge extends Component {
+	static propTypes = {
+		siteSlug: PropTypes.string.isRequired,
+		trackNudgeAlreadyListedClick: PropTypes.func.isRequired,
+		trackNudgeDismissClick: PropTypes.func.isRequired,
+		trackNudgeStartNowClick: PropTypes.func.isRequired,
+		trackNudgeView: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	componentWillMount() {
+		if ( ! this.props.isDismissed ) {
+			this.props.trackNudgeView();
+		}
+	}
+
+	render() {
+		if ( this.props.isDismissed ) {
+			return null;
+		}
+
+		return (
+			<Card className="google-my-business-stats-nudge">
+				<QueryPreferences />
+				<Gridicon
+					icon="cross"
+					className="google-my-business-stats-nudge__close-icon"
+					onClick={ flow( this.props.trackNudgeDismissClick, this.props.dismissNudge ) }
```

Comment:
Each render creates a new function here, maybe move to c-tor?

![screen shot 2018-02-26 at 15 59 39](https://user-images.githubusercontent.com/326402/36674240-2d069d1c-1b0e-11e8-8024-db2a9d948b3f.png)